### PR TITLE
Hungarian language support

### PR DESCRIPTION
**Pull request by blackPantherOS**
Hi! We added initial Hungarian support and pre-translated files

### CODE REVIEW

File: spyder/config/base.py [hunk 1]
```diff
@@ -229,6 +229,7 @@ def get_image_path(name, default="not_found.png"):
 LANGUAGE_CODES = {'en': u'English',
                   'fr': u'Français',
                   'es': u'Español',
+                  'hu': u'Hungarian',
```
Comment (goanpeca) [inline spyder/config/base.py]: Thanks a lot for this :-)

Now, the idea here is to use the Native language name, I am guessing `Magyar` ?

Comment (ccordoba12) [inline spyder/config/base.py]: Yep, please use here the name for Hungarian in Hungarian itself.

**Comment by ccordoba12** [comment]
@blackPantherOS, thanks a lot for this!! It's a great contribution.

Unfortunately, since you did your translation against our `master` branch, it will be available when we release Spyder **4**, and not in any of our **3.x** releases.

I hope that's ok for you.

File: spyder/config/base.py (hunk 1 above)
Comment (blackPantherOS) [inline spyder/config/base.py]: Hungarian in English and Magyar in Hungarian :) I think fine both if either available

Comment (ccordoba12) [inline spyder/config/base.py]: Our convention is to use the native language name, as you can see with the other languages.

So please make our requested change.

**Comment by ccordoba12** [comment]
@blackPantherOS, one last question: would you be available in the future to keep updating this translation?

**Comment by goanpeca** [comment]
@martindurant want to help ?

**Comment by blackPantherOS** [comment]
@ccordoba12 Sorry because 
I'm too late that I write any response. We have more thousand other project in blackPanther OS. Not have too many time for one project and translation, time to time we would like to help you

**Comment by ccordoba12** [comment]
Closing in favor of #4473. @blackPantherOS, don't worry about it, your work will be available in Spyder 3.2!

**Pull request closed without merging**